.github/workflows/ci.yml

```diff
@@ -148,9 +108,9 @@ jobs:
         if: matrix.os == 'ubuntu-latest'
         run: cargo test --workspace --all-features
 
-      - name: Cargo build (all workspace without elements-fuzz )
+      - name: Cargo build
         if: matrix.os == 'macos-latest'
-        run: cargo build --workspace --exclude elements-fuzz --all-features
+        run: cargo build --workspace --all-features
```

Comment:
This can be removed now together with line 108.

Reply:
If I remove what you suggest we run docker tests on MacOS which [fails](https://github.com/comit-network/waves/pull/134/checks?check_run_id=1859103372#step:5:492). Did I misunderstand you?

Reply:
Ah yeah, I am an idiot. Mixed up build and test script :(